.taskcluster.yml

```diff
@@ -24,9 +24,15 @@ tasks:
           go get -t github.com/taskcluster/taskcluster-cli/... 
           && cd  /go/src/github.com/taskcluster/taskcluster-cli 
           && git fetch {{ event.head.repo.url }} {{ event.head.ref }} 
-          && git checkout {{ event.head.sha }} 
           && make 
-          && go test ./...
+          && go get -u github.com/alecthomas/gometalinter 
+          && gometalinter --install
+          && go get -u ./...
+          && git checkout {{ event.head.sha }} 
```

Comment:
@djmitche nope, tests run under vendor, just `gometalinter`

@t0xicCode oh, I was not aware of vendor option, thanks. @lteigrob could you please give it a try?